canonical/canonical.com
# Bump global-nav version

## Done

- Bumped global nav version
- Added some custom CSS

## QA

- View the site locally in your web browser at: http://0.0.0.0:8002/data (or any other page that has the global nav)
- See that the links are no longer blacked out


## Issue / Card

Fixes #

## Screenshots

[if relevant, include a screenshot]
